site-root/sass/slate/modules/_omnibar.scss

```diff
@@ -112,6 +112,7 @@
 .omnibar-search-item {
     @include flex(1);
     margin: 0 1em 0 .5em;
+    cursor: pointer;
```

Comment:
this should only be set on the field, otherwise the empty space that is not clickable to the left of it also gets the pointer cursor